src/v/cluster/rm_stm.cc

```diff
@@ -2784,6 +2805,8 @@ rm_stm::load_abort_snapshot(abort_index index) {
 
     iobuf_parser data_parser(std::move(data_buf));
     auto data = reflection::adl<abort_snapshot>{}.from(data_parser);
+    _abort_snapshot_size += co_await _abort_snapshot_mgr.get_snapshot_size(
```

Comment:
Is `load_abort_snapshot` called only on startup? It is a bit strange but I think not - it is also called in `aborted_transactions` which means that we can count some snapshot sizes several times, is that true?